mediaire_toolbox/queue/tasks.py

```diff
@@ -61,16 +74,22 @@ def read_dict(self, d):
         tag = d['tag']
         timestamp = d['timestamp']
```

Comment:
Why are these fields optional in `Task.__init__` but not here? I think it should be consistent...